CHANGELOG

```diff
@@ -1,71 +1,80 @@
-0.5.0 2019-11-25)
```

Comment:
Oh finally! I wanted to change this forever, but kept the original formatting just because I'm lazy.